### PR TITLE
add helm config variables for dev environment

### PR DESCRIPTION
**Pull request by rgforsyth**
## What does this pull request do?

Adds the configuration variables so that the intervention service can send a request to the community-api in the dev region.

## What is the intent behind these changes?

To allow the application to work in the dev region.

### CODE REVIEW

File: helm_deploy/hmpps-interventions-service/templates/_envs.tpl [hunk 1]
```diff
@@ -19,6 +19,33 @@ env:
   - name: INTERVENTIONSUI_BASEURL
     value: "{{ .Values.env.INTERVENTIONSUI_BASEURL }}"
 
+  - name: COMMUNITYAPI_BASEURL
+    value: "{{ .Values.env.COMMUNITYAPI_BASEURL }}"
+
+  - name: COMMUNITYAPI_TEMPORARYREFERRALREQUEST_PROBATIONAREACODE
```
Comment (tomsmyers) [inline helm_deploy/hmpps-interventions-service/templates/_envs.tpl:25]: naming is incorrect on these -> contactNotificationContext,

same below

Comment (rgforsyth) [reply helm_deploy/hmpps-interventions-service/templates/_envs.tpl]: Fixed

**Review by sldblog — APPROVED** [review]
Looks good to me 👍 No blockers.

Observation: making the context configuration required pushes the responsibility of configuring it to whoever wants to run the service (either via docker-compose, directly from docker, minikube/local kubernetes, future integration tests on CircleCI, etc.)

Overall, no one _outside_ our service will know how to configure these values. They could figure out base URLs and client IDs, but this is domain-specific.

It's totally not a blocker though

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -18,3 +18,8 @@ env:
   JAVA_OPTS: "-Xmx512m"
   HMPPSAUTH_BASEURL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   INTERVENTIONSUI_BASEURL: "https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk"
+  COMMUNITYAPI_BASEURL: "https://community-api-secure.test.delius.probation.hmpps.dsd.io"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_PROBATIONAREACODE: "A00"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_REFERRALTYPE: "C385"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_STAFFCODE: "ASPUATU"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE: "ASPUAT"
```
[the remark this answers is not in the document]
Comment (rgforsyth) [reply helm_deploy/values-dev.yaml:25]: Agreed. 
At some point I believe these may be removed completely and grabbed from somewhere. Until it is decided where/what the plan is, it was recommended to add them here.

**Comment by tomsmyers** [comment]
> Looks good to me 👍 No blockers.
> 
> Observation: making the context configuration required pushes the responsibility of configuring it to whoever wants to run the service (either via docker-compose, directly from docker, minikube/local kubernetes, future integration tests on CircleCI, etc.)
> 
> Overall, no one _outside_ our service will know how to configure these values. They could figure out base URLs and client IDs, but this is domain-specific.
> 
> It's totally not a blocker though

these are also set in 'local' config, so you can run this via the methods you mentioned here

**Pull request merged**